scheduler/roles/cluster_validation/tasks/fetch_omnia_inputs.yml

```diff
@@ -77,19 +77,19 @@
   when:
     - slurm_installation_type | default("", true) not in ["nfs_share", "configless"]
 
-- name: Assert mariadb_password
-  ansible.builtin.assert:
-    that:
-      - mariadb_password | default("", true) | length > 1
-      - mariadb_password | length > min_length | int - 1
-      - mariadb_password | length < max_length | int + 1
-      - '"-" not in mariadb_password '
-      - '"\\" not in mariadb_password '
-      - '"\"" not in mariadb_password '
-      - " \"'\" not in mariadb_password "
-    success_msg: "{{ success_msg_mariadb_password }}"
-    fail_msg: "{{ fail_msg_mariadb_password }}"
-  when: slurm_support
+# - name: Assert mariadb_password
+#   ansible.builtin.assert:
+#     that:
+#       - mariadb_password | default("", true) | length > 1
+#       - mariadb_password | length > min_length | int - 1
+#       - mariadb_password | length < max_length | int + 1
+#       - '"-" not in mariadb_password '
+#       - '"\\" not in mariadb_password '
+#       - '"\"" not in mariadb_password '
+#       - " \"'\" not in mariadb_password "
+#     success_msg: "{{ success_msg_mariadb_password }}"
+#     fail_msg: "{{ fail_msg_mariadb_password }}"
+#   when: slurm_support
 
 - name: Verify the value of restart_slurm_services
```

Comment:
missing validation of new variables

Reply:
the varibales have default values, its handled